test/IdRegistry/IdRegistry.t.sol

```diff
@@ -300,6 +300,74 @@ contract IdRegistryTest is IdRegistryTestSuite {
         assertEq(idRegistry.getRecoveryOf(1), address(0));
     }
 
+    function testFuzzVerifyFidSignature(
+        address registrar,
+        uint256 recipientPk,
+        address recovery,
+        uint40 _deadline
```

Comment:
we don't need to fuzz `registrar`, `_deadline` or `recovery` here and we should be fuzzing `digest` in addition to `recipientPk`